vnext/Shared/AsyncStorage/AsyncStorageManager.cpp

```diff
@@ -104,7 +104,7 @@ void AsyncStorageManager::executeAsyncKVOperation(
         break;
 
       default:
-        [this, jsCallback] { jsCallback({makeError("Invalid AsyncStorage operation")}); };
+        [this, jsCallback] { jsCallback({makeError("Invalid AsyncStorage operation")}); }();
```

Comment:
Seems like you could just remove the lambda entirely here and call jsCallack directly.

Reply:
@NickGerleman you're not wrong... I tried looking up where this came from but it seems like it always was this way 🤷‍♀️ fixing it to just call the callback